scripts/create_incr_tickets.py

```diff
@@ -163,14 +163,17 @@ def crawl(path, TARGET_FILE_NUMBER):
 
 def main():
     path = sys.argv[1]
+    ticket_number_seed = int(sys.argv[2])
     batches = crawl(path, TARGET_FILE_NUMBER)
     with open('output.csv', 'w') as out:
-        out.write('BLOCKED, NUMBER OF PYTHON FILES, DIRECTORIES')
+        out.write('INCR-TICKET, BLOCKED, NUMBER OF PYTHON FILES, DIRECTORIES')
         out.write('\n')
         for b in batches:
             dirs = ':'.join(b.top_level_directories)
-            out.write('{},{},{}'.format(b.blocked, b.file_count(), dirs))
+            ticket_number = "INCR-{}".format(ticket_number_seed)
```

Comment:
Should we skip allocating ticket numbers to blocked batches?

Reply:
Ah, yeah, so I wasn't sure about the best way to handle that. One thing to do is to create the branch, but don't create a PR (so no tests are run) for blocked tickets. What do you think @jmbowman